src/Stratis.Bitcoin.Tests/Utilities/DateTimeProviderTest.cs

```diff
@@ -7,28 +7,40 @@ namespace Stratis.Bitcoin.Tests.Utilities
 {
     public class DateTimeProviderTest
     {
+        /// <summary>
+        /// Because we can not make multiple calls to DateTime.UtcNow and guarantee that the result is the same,
+        /// we need to check that the returned value falls within a given threshold period. We arbitrarily choose an interval
+        /// and assume that the subsequent calls to DateTime.UtcNow will always take less time than that.
+        /// </summary>
+        public const int TimeIntervalThreshholdSeconds = 2;
```

Comment:
This should `TimeIntervalThresholdSeconds`

Reply:
```suggestion
        public const int TimeIntervalThresholdSeconds = 2;
```
;)